@ndlib/gatsby-theme-marble/src/assets/theme.js

```diff
@@ -3,7 +3,7 @@
 * https://github.com/system-ui/theme-ui/blob/develop/packages/preset-bootstrap/src/index.ts
 */
 
-import merge from 'lodash.merge'
+import { merge } from 'theme-ui'
```

Comment:
Same as other, can we remove lodash.merge from package.json?

Reply:
yeah i will do that

Reply:
actually no it is still used in the i18n merge.